src/components/LanguageEngagementListItemCard/LanguageEngagementListItemCard.tsx

```diff
@@ -79,9 +79,8 @@ export const LanguageEngagementListItemCard: FC<LanguageEngagementListItemCardPr
   const language = props.language.value;
   const name = language?.name.value ?? language?.displayName?.value;
   const population =
-    language?.organizationPopulation?.value ??
-    language?.ethnologuePopulation?.value;
-  const rodNumber = language?.rodNumber?.value;
+    language?.population.value ?? language?.ethnologue.population.value;
+  const rodNumber = language?.registryOfDialectsCode.value;
```

Comment:
I'd rather not leave "rodNumber" hanging around. All of those instances should be replaced.

Reply:
Removed